javascript/javascript1/week2/week2-guzide/event-application/EventApp.js

```diff
@@ -0,0 +1,19 @@
+const weekDaysArr = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+
+function getEventWeekday(daysFromNow) {
+  const todaysIndex = new Date().getDay();
+  const eventDaysIndex = (todaysIndex + daysFromNow) % 7;
+  return console.log(weekDaysArr[eventDaysIndex]);
```

Comment:
Very nice use of modulus!
You do not need the keyword `return` before console.log. It would make more sense to return the actual weekday from this function, then the value can be stored in a varialbe and the fuction will be more reusable. Try to refactor the code that way.

Reply:
The function is refactored.